### ISSUE TITLE
No support for macOS 10.13 and will not get edited settings

### ISSUE THREAD

**Issue by ppap9264** [vincss/mcsleepingserverstarter#212]
Is there any version of this for a mac running macOS 10.13?

The other issue is that the executable file will not read the settings from settings.yml, even after i edited it and saved it as its original form.

**Comment by vincss** [comment]
I got a virtual box running in 10.15.7 and it's running fine.

![image](https://github.com/vincss/mcsleepingserverstarter/assets/2254806/3075abfc-d94f-4639-9e7d-43cc5e1c227d)

What's your issue ?

**Comment by ppap9264** [comment]
Yeah, it runs fine on 10.15.X, but on 10.13.X it doesnt seem to work. I have put a screenshot below. 
![Screen Shot 2023-11-10 at 7 33 17 AM](https://github.com/vincss/mcsleepingserverstarter/assets/110489671/20a41a76-5e05-4d37-aec9-b0b5b31c1782)

**Comment by ppap9264** [comment]
Any updates?

**Comment by vincss** [comment]
Hi !
 I don't know much about MacOs and how to build binary for it.
Maybe you can give a try directly from the source : https://github.com/vincss/mcsleepingserverstarter/tree/master#-manually-install-and-compile

**Comment by ppap9264** [comment]
I think it may be a problem with the node protocol. to compile from the source, is it just npm install?

**Comment by vincss** [comment]
You just have to do: 

- git clone git@github.com:vincss/mcsleepingserverstarter.git
- cd mcsleepingserverstarter
- npm ci
- npm start
And tell me what error you have.

**Comment by ppap9264** [comment]
Alright, thanks a lot. Will do that today and will get back to you.

**Comment by ppap9264** [comment]
Good news. I have updated to macOS Montery and patched it. The sleepingserverstarter works very well, except for a 1 things.

1: When I connect to the server, i get an error (attached on the bottom).
<img width="1704" alt="Screenshot 2023-11-17 at 6 05 35 PM" src="https://github.com/vincss/mcsleepingserverstarter/assets/110489671/eabd7c87-5411-41f6-a535-f8d88ab82cef">

**Comment by vincss** [comment]
Yes it's linked to this issue, we are waiting for an update of the minecraft protocol library : 
https://github.com/vincss/mcsleepingserverstarter/issues/200
